repos.d/adelie.yaml

```diff
@@ -0,0 +1,20 @@
+###########################################################################
+# Adélie Linux aports
+###########################################################################
+- name: adelie_current
+  type: repository
+  desc: Adélie Linux current
+  family: alpine
+  ruleset: alpine
+  color: '#880000'
+  minpackages: 900
+  sources:
+    - name: [ system, user ]
+      fetcher: WgetTarFetcher
+      parser: ApkIndexParser
+      url: 'https://distfiles.adelielinux.org/adelie/current/{source}/ppc64/APKINDEX.tar.gz'
+      subrepo: '{source}'
+  repolinks:
+    - desc: Adélie Linux home
+      url: https://adelielinux.org/
+  tags: [ all, production ]
```

Comment:
We're still in beta.  is 'production' the correct tag?

Reply:
I took this to mean that it is available on repology.org. @AMDmi3 thoughts?

Reply:
This of course relates to the status of repository in Repology - `production` denotes that the repository is available on [repology.org](https://repology.org) instance. We don't care of upstream production status.